helix-view/src/editor.rs

```diff
@@ -689,7 +689,7 @@ pub struct WhitespaceCharacters {
 impl Default for WhitespaceCharacters {
     fn default() -> Self {
         Self {
-            space: '·',    // U+00B7
+            space: '·',   // U+00B7
```

Comment:
This was recently addressed in #7013 so if you rebase then this line should disappear from the diff

Reply:
I don't think it was changed, I'll save the file without formatting that line if you want